examples/framework/preferences/localizationPreference/urlPath/js/localization.js

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2018 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid */
+
+var example = example || {};
+(function ($, fluid) {
+    "use strict";
+
+    fluid.prefs.builder({
+        gradeNames: ["fluid.prefs.auxSchema.localization"],
+        auxiliarySchema: {
+            "namespace": "example.prefs.localization"
+        }
+    });
+
+    fluid.defaults("example.prefsEditor", {
```

Comment:
I'm not sure how useful this examplary code will help integrators. With all these uses of aux schema namespace, distributeOptions targets etc, writing this level of grade requires a good understanding of the detailed implementation of the prefs framework schema version. But of course, it's great to demonstrate how the localization can be done.

Reply:
hmm.. that's interesting.. perhaps there is a helper grade missing that should include many of those things and could be added directly to the schema. I'll have to investigate that.